ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -76,3 +76,24 @@ if [ -n "${FEATURE_GATES}" ]; then
 featureGates: ${FEATURE_GATES}
 EOF
 fi
+
+# Set image-registry replicas to 3; today, we will add in the anti-affinity configuration
+# because the image-registry operator only adds this when the replicas is set to 2.
```

Comment:
Could we check for an ENVAR and only output this when present, then configure the serial job to set the ENVAR?